src/main/resources/public/json/profiles.json

```diff
@@ -2,7 +2,10 @@
   "profiles": {
     "123456789": {
       "link": "https://github.com/123456789",
```

Comment:
link should also depend on the year. I may use different repositories

Reply:
I don't think so. It is just a link to their github's/other interface's profile

Reply:
Nah, users tend to link to their repo and make a new repo each year

Reply:
Ok it is resolved from now